src/portal/src/i18n/lang/fr-fr-lang.json

```diff
@@ -1632,181 +1629,181 @@
         "DELETE_ACCESSORY": "Supprimer l'accessoire",
         "DELETED_SUCCESS": "Accessoire supprimé avec succès",
         "DELETED_FAILED": "Echec de la suppression de l'accessoire",
-        "CO_SIGNED": "Signé by Cosign",
-        "NOTARY_SIGNED": "Signé by Notary",
+        "CO_SIGNED": "Signé par Cosign",
+        "NOTARY_SIGNED": "Signé par Notary",
         "ACCESSORY": "Accessoire",
         "ACCESSORIES": "Accessoires",
-        "SUBJECT_ARTIFACT": "Sujet de l'Artefact",
+        "SUBJECT_ARTIFACT": "Sujet de l'artefact",
         "CO_SIGN": "Cosign",
         "NOTARY": "Notary",
         "PLACEHOLDER": "Nous n'avons trouvé aucun accessoire !"
     },
     "CLEARANCES": {
-        "CLEARANCES": "Clean Up",
-        "AUDIT_LOG": "Log Rotation",
-        "LAST_COMPLETED": "Last completed",
-        "NEXT_SCHEDULED_TIME": "Next scheduled time",
-        "SCHEDULE_TO_PURGE": "Schedule to purge",
-        "KEEP_IN": "Keep records in",
-        "KEEP_IN_TOOLTIP": "Keep the records in this interval",
-        "KEEP_IN_ERROR": "The value of this item must be an integer, and the time value must be greater than 0 and less than 10000 days",
-        "DAYS": "Days",
-        "HOURS": "Hours",
-        "INCLUDED_OPERATIONS": "Included operations",
-        "INCLUDED_OPERATION_TOOLTIP": "Remove audit logs for the selected operations",
-        "INCLUDED_OPERATION_ERROR": "Please select at lease one operation",
-        "PURGE_NOW": "PURGE NOW",
-        "PURGE_NOW_SUCCESS": "Purge triggered successfully",
-        "PURGE_SCHEDULE_RESET": "Purge schedule has been reset",
-        "PURGE_HISTORY": "Purge History",
-        "FORWARD_ENDPOINT": "Audit Log Forward Syslog Endpoint",
-        "FORWARD_ENDPOINT_TOOLTIP": "Forward audit logs to the syslog endpoint, for example: harbor-log:10514",
-        "SKIP_DATABASE": "Skip Audit Log Database",
-        "SKIP_DATABASE_TOOLTIP": "Skip to log audit log in the database, only available when audit log forward endpoint is configured",
-        "STOP_GC_SUCCESS": "Trigger stopping GC operation successfully",
-        "STOP_PURGE_SUCCESS": "Trigger stopping purging operation successfully",
-        "NO_GC_RECORDS": "We couldn't find any GC histories!",
-        "NO_PURGE_RECORDS": "We couldn't find any purge histories!"
+        "CLEARANCES": "Service de nettoyage",
```

Comment:
Maybe "archivage" instead of "nettoyage" ? Or purge ?

Reply:
In english they translated it into "Clean up". It's used in file `clearing-job.component.html`.
Maybe purge would be more appropriate